src/components/Confirm/CaptureViewer/style.css

```diff
@@ -50,6 +50,16 @@
   object-fit: contain;
 }
 
+.croppedDocumentImage {
+  width: 100%;
+  height: 13.5em;
+  object-fit: cover;
+  object-position: center;
+  @media (--small-viewport) and (orientation: landscape) {
+    max-height: 310px;
```

Comment:
How is the UI looking on small screens like iPhone 5 and iPhone SE. Also why is the value in `px`?

Reply:
Also I think that this class should behave in the same way as the `.image` class where the image becomes smaller and smaller depending on the screen size. That way the text is alway visible on any screen. Again, might be worth testing this on iPhone 5 on browserstack and make sure it behaves well also when there is a validation error

Reply:
My bad. I shouldn't be using a pixel value for that. Or limiting to just landscape orientation.

In the case that this class is made to behave like `.image` class, we'd lose the cropped preview though. Or maybe I'm misunderstanding what you mean?

Reply:
I meant it should behave like `.image` in terms of responsiveness so using `max-height: 100%;` here will probably do the trick

Reply:
ah, ok. gotcha. 👍